esx_service/vmci/vmci_client.c

```diff
@@ -161,9 +168,10 @@ dummy_get_reply(be_sock_id *id, be_request *r, be_answer* a)
 static be_sock_status
 vsock_init(be_sock_id *id, int cid, int port)
 {
+   static int round_robin = START_CLIENT_PORT; // Round robin client bind port
```

Comment:
I liked Mark's original suggestion to name it as "client_port ". Comment takes care of explaining port is selected in round robin fashion.